configs/dest/README.md

```diff
@@ -0,0 +1,49 @@
+
+<!-- [IMAGE] -->
+
+<div align=center>
+<img src="https://github.com/NVIDIA/DL4AGX/blob/master/DEST/resources/attentions.png" width="70%"/>
+</div>
+
+## Citation
+
+```bibtex
+@article{YangDEST,
+  title={Depth Estimation with Simplified Transformer},
+  author={Yang, John and An, Le and Dixit, Anurag and Koo, Jinkyu and Park, Su Inn},
+  journal={arXiv preprint arXiv:2204.13791},
+  year={2022}
+}
+```
+
+## Results and models
+
+### Cityscapes
+
+
+| Method    | Backbone | Crop Size | Lr schd | Mem (GB) | Inf time (fps) |  mIoU | mIoU(ms+flip) | config                                                                                                                                 | download                                                                                                                                                                                                                                                                                                                                                                                       |
+| --------- | -------- | --------- | ------: | -------: | -------------- | ----: | ------------- | -------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| DEST | MIT-B0   | 1024x1024 |  160000 |     - | -           | 64.34 | -         | - | - | - |
+| DEST | MIT-B1   | 1024x1024 |  160000 |     - | -            | 68.21 | -        | - | - | - |
+| DEST | MIT-B2   | 1024x1024 |  160000 |     - | -           | 71.89 | -         | - | - | - |
+| DEST | MIT-B3   | 1024x1024 |  160000 |    - | -           | 73.51 | -         | - | - | - |
+| DEST | MIT-B4   | 1024x1024 |  160000 |    - | -           | 73.99 | -         | - | - | - |
+| DEST | MIT-B5   | 1024x1024 |  160000 |    - | -          | 75.28 | -        | - | - | - |
+
+
+- Training of DEST is not very stable, which is sensitive to random seeds.
```

Comment:
Add a sub heading of "Note:"
Add a note about training from scratch, not with a pretrained model
Add link to config, model/log

Reply:
model/logs are under: https://nvidia-my.sharepoint.com/:f:/p/boyinz/EqOEo1BZ1V9LpMKLoldH4YkBRvG4xWqhJSUdUVah7VhqDw?e=WB45AM

Reply:
nvidia-my.sharepoint.com is not accessible from outside, can you move to your personal google drive or dropbox?

Reply:
https://drive.google.com/drive/folders/1MOnW41XKPRYseOtxh_wgsbgpt7xwmeLb?usp=sharing